cli.js

```diff
@@ -61,17 +63,32 @@ const run = (input, opts) => {
 	opts = Object.assign({plugin: DEFAULT_PLUGINS}, opts);
 
 	const use = requirePlugins(arrify(opts.plugin));
+	const spinner = ora('Minifying images');
 
 	if (Buffer.isBuffer(input)) {
 		imagemin.buffer(input, {use}).then(buf => process.stdout.write(buf));
 		return;
 	}
 
-	imagemin(input, opts.outDir, {use}).then(files => {
-		if (!opts.outDir) {
-			files.forEach(x => process.stdout.write(x.data));
-		}
-	});
+	if (opts.outDir) {
+		spinner.start();
+	}
+
+	imagemin(input, opts.outDir, {use})
+		.then(files => {
+			if (!opts.outDir) {
+				files.forEach(x => process.stdout.write(x.data));
```

Comment:
Unrelated to this PR, but this makes no sense. You can't output multiple images to stdout as there's no separator, so no tool would be able to handle it. I think we instead should throw an helpful error about this if `outDir` is not specified and the input is more than one file.